tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.sessions.middleware import SessionMiddleware
+from django.test import RequestFactory
+
+from factories import (UserFactory, InstitutionFactory)
+
+
+@pytest.fixture
+def user(uai_number):
+    user = UserFactory(
+        id=1,
+        email='noel@flantier.com',
+        is_active=True
+    )
+    InstitutionFactory(
+        uai=uai_number,
+        institution_name='Lycée John, Doe',
+        user_id=1
```

Comment:
If we are not fixing the id for the user, couldn't we use
```python
user=user
```
instead

Reply:
Fixed by a37b1fd